devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -312,7 +312,18 @@ def zwaveEvent(DoorLockOperationReport cmd) {
 			result << response(secure(zwave.associationV1.associationGet(groupingIdentifier:1)))
 		}
 	}
-	result ? [createEvent(map), *result] : createEvent(map)
+	result = result ? [createEvent(map), *result] : createEvent(map)
```

Comment:
`createEvent` sets `isStateChange`. It would be better to move this line to the `else` block below and just send `map` to `delayLockEvent`.

Reply:
Good catch @mckeed